sound/hda/intel-dsp-config.c

```diff
@@ -293,26 +293,13 @@ static const struct config_entry config_table[] = {
 	{
 		.flags = FLAG_SOF,
 		.device = 0xa0c8,
-		.dmi_table = (const struct dmi_system_id []) {
-			{
-				.ident = "Google Chromebooks",
-				.matches = {
-					DMI_MATCH(DMI_SYS_VENDOR, "Google"),
-				}
-			},
-			{}
-		}
-	},
-	{
-		.flags = FLAG_SOF | FLAG_SOF_ONLY_IF_DMIC_OR_SOUNDWIRE,
-		.device = 0xa0c8,
```

Comment:
No that's not correct. If we don't have DMIC/SoundWire, and it's not a Chromebook, then we want to use the legacy HDaudio driver.
The Chromebook does not rely on NHLT so the DMIC detection will fail, so it needs a dedicated quirk.

This needs to be reverted IMHO.